_rules/heading-accessible-name-ffd0e9.md

```diff
@@ -0,0 +1,146 @@
+---
+id: ffd0e9
+name: Heading has accessible name
+rule_type: atomic
+description: |
+  This rule checks that each heading has an accessible name.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
```

Comment:
I think this simplified version would be relevant for 2.4.6 as well as 1.3.1, as a heading cannot describe something if it is empty.